iothub/device/src/Transport/ErrorDelegatingHandler.cs

```diff
@@ -36,7 +35,7 @@ public ErrorDelegatingHandler(IPipelineContext context, IDelegatingHandler inner
             typeof(OperationCanceledException),
             typeof(HttpRequestException),
             typeof(WebException),
-            typeof(AmqpException),
+            typeof(IotHubCommunicationException),
```

Comment:
Please revert. We already have proper handling for IotHubCommunicationException. This is here to avoid any exception somehow thrown by the stack directly.